src/controllers/PointModeController.js

```diff
@@ -175,6 +176,11 @@ class PointModeController extends AnnotationModeController {
     pointClickHandler(event) {
```

Comment:
Is this the handler to enter point mode? Or to drop an annotation?

Reply:
to enter point annotation mode